aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -358,11 +362,11 @@ def parse_scheduler_output(self, retrieved):
 
         try:
             exit_code = scheduler.parse_output(detailed_job_info, scheduler_stdout, scheduler_stderr)
-        except exceptions.FeatureNotAvailable as exception:
-            self.logger.warning('could not parse scheduler output: {}'.format(exception))
+        except exceptions.FeatureNotAvailable:
+            self.logger.info('`{}` does not implement scheduler output parsing'.format(scheduler.__class__.__name__))
             return
         except Exception as exception:  # pylint: disable=broad-except
-            self.logger.warning('the `parse_output` method of the scheduler excepted: {}'.format(exception))
+            self.logger.error('the `parse_output` method of the scheduler excepted: {}'.format(exception))
```

Comment:
Would it be easy to add a test that tries to catch this exception? You would have to monkeypatch the parser/calcjob?

Reply:
Yeah, should be easy enough to monkey path the parser, but checking logs can be a bit trickier. But I will give it a shot.